src/policy/policy.h

```diff
@@ -53,6 +53,8 @@ static constexpr unsigned int DEFAULT_WEIGHT_PER_DATA_BYTE{4};
 static constexpr bool DEFAULT_PERMIT_BAREPUBKEY{false};
 /** Default for -permitbaremultisig */
 static constexpr bool DEFAULT_PERMIT_BAREMULTISIG{false};
+/** Default for -rejectparasites */
+static constexpr bool DEFAULT_REJECT_PARASITES{true};
```

Comment:
Make this `false` for the first commit and add a third with it where `corepolicy` is also updated (eg https://github.com/bitcoinknots/bitcoin/commit/58c1e5ce547d61760a41e0febf68690c8e55d4bb)